### PR TITLE
RRD: replace DOCUMENT_TITLE constant with PREFIX and SUFFIX constants

### PR DESCRIPTION
**Pull request by yoomlam**
## Description of change

Replace `DOCUMENT_TITLE` constant with `DOCUMENT_NAME_PREFIX` and `DOCUMENT_NAME_SUFFIX` constants.

## Original issue(s)
`DOCUMENT_TITLE` is specific to the hypertension disability, so we can't use it for other disabilities.

### CODE REVIEW

File: spec/services/rapid_ready_for_decision/fast_track_pdf_upload_manager_spec.rb [hunk 1]
```diff
@@ -4,27 +4,7 @@
...
-
-  let(:form526_submission) do
-    Form526Submission.create(
-      user_uuid: user.uuid,
-      saved_claim_id: saved_claim.id,
-      auth_headers_json: auth_headers.to_json,
-      form_json: original_form_json
-    )
-  end
+  let(:form526_submission) { create(:form526_submission, :hypertension_claim_for_increase_with_uploads) }
```
Comment (yoomlam) [inline spec/services/rapid_ready_for_decision/fast_track_pdf_upload_manager_spec.rb:7]: Replace all this with a FactoryBot call.

File: spec/services/rapid_ready_for_decision/fast_track_pdf_upload_manager_spec.rb [hunk 2]
```diff
@@ -51,9 +36,7 @@
       end
 
       context 'there are no existing uploads present in the list' do
-        let(:original_form_json) do
-          File.read('spec/support/disability_compensation_form/submissions/without_form526_uploads.json')
-        end
+        let(:form526_submission) { create(:form526_submission, :hypertension_claim_for_increase) }
```
Comment (yoomlam) [inline spec/services/rapid_ready_for_decision/fast_track_pdf_upload_manager_spec.rb:39]: Trait `:hypertension_claim_for_increase` has no uploads specified.

File: spec/services/rapid_ready_for_decision/fast_track_pdf_upload_manager_spec.rb (hunk 1 above)
Comment (leikkisa) [reply spec/services/rapid_ready_for_decision/fast_track_pdf_upload_manager_spec.rb:7]: 🎉

**Review by leikkisa — APPROVED** [review]
Looks great!

**Review by RachalCassity — APPROVED** [review]

**Pull request merged**